sys/arm64/arm64/exception.S

```diff
@@ -60,22 +60,20 @@ __FBSDID("$FreeBSD$");
 	stp	CAP(4),  CAP(5),  [PTRN(sp), #(TF_X + 4  * CAP_WIDTH)]
 	stp	CAP(2),  CAP(3),  [PTRN(sp), #(TF_X + 2  * CAP_WIDTH)]
 	stp	CAP(0),  CAP(1),  [PTRN(sp), #(TF_X + 0  * CAP_WIDTH)]
-#if __has_feature(capabilities)
-	mrs	c9, ddc_el0
```

Comment:
Batching the MRSes up before the STR/STPes is generally better instruction scheduling and helps reduce stalls, so I disagree with this change. Similarly for the restore path with the LDR/LDPes being batched up before the MSRes.